File: C vs Python/c prog/2.cpp
//This does the age comparison of two person by first taking age as inputs from user
#include <iostream>
using namespace std;

main()
{

float waqas,waqar;	
	
cout << "Please enter the age of Waqas ";
cin >> waqas;
cout << "Please enter the age of Waqar ";
cin >> waqar;

if (waqas > waqar)
{
	cout << "Waqas is older than Waqar";
}
else if (waqas < waqar)
{
	cout << "Waqas is younger than Waqar";
}
else
{
	cout << "Waqas is same as Waqar";
}

}

File: C vs Python/py prog/2.py
#This does the age comparison of two person by first taking age as inputs from user

waqas = float(input("Please enter the age of Waqas "))
waqar = float(input("Please enter the age of Waqar "))


if (waqas > waqar):
    print ("Waqas is older than Waqar")
elif (waqas < waqar):
    print ("Waqas is younger than Waqar")
else:
    print ("Waqas is same as Waqar")